aries-site/package.json

```diff
@@ -23,7 +23,7 @@
     "start": "next",
     "start-server": "yarn build && yarn next start -p 3030",
     "test": "yarn build && node src/scripts/start-tests.js",
-    "test:local": "start-server-and-test start-server 3030 'testcafe all src/tests/'",
+    "test:local": "start-server-and-test start-server 3030 'testcafe chrome,firefox src/tests/'",
```

Comment:
I might be missing something shouldn't `test:local`  be added to the package.json pre-commit section to perform as a pre-commit?

Reply:
`test:local` is called by `huskyrc.js` -- it is an alternative to the `pre-commit` package.